pkg/remote/azurerm_privatedns_scanner_test.go

```diff
@@ -26,55 +31,210 @@ func TestAzurermPrivateDNSZone(t *testing.T) {
 	dummyError := errors.New("this is an error")
 
 	tests := []struct {
-		test           string
-		mocks          func(*repository.MockPrivateDNSRepository, *mocks.AlerterInterface)
-		assertExpected func(t *testing.T, got []*resource.Resource)
-		wantErr        error
+		test    string
+		dirName string
+		mocks   func(*repository.MockPrivateDNSRepository, *mocks.AlerterInterface)
+		wantErr error
 	}{
 		{
-			test: "no PrivateZone",
+			test:    "no private zone",
+			dirName: "azurerm_private_dns_private_zone_empty",
 			mocks: func(repository *repository.MockPrivateDNSRepository, alerter *mocks.AlerterInterface) {
 				repository.On("ListAllPrivateZones").Return([]*armprivatedns.PrivateZone{}, nil)
 			},
-			assertExpected: func(t *testing.T, got []*resource.Resource) {
-				assert.Len(t, got, 0)
-			},
 		},
 		{
-			test: "error listing PrivateZone",
+			test:    "error listing private zones",
+			dirName: "azurerm_private_dns_private_zone_empty",
 			mocks: func(repository *repository.MockPrivateDNSRepository, alerter *mocks.AlerterInterface) {
 				repository.On("ListAllPrivateZones").Return(nil, dummyError)
 			},
-			wantErr: error2.NewResourceListingError(dummyError, resourceazure.AzurePrivateDNSZoneResourceType),
+			wantErr: remoteerr.NewResourceListingError(dummyError, resourceazure.AzurePrivateDNSZoneResourceType),
 		},
 		{
-			test: "multiple PrivateZone",
+			test:    "multiple private zones",
+			dirName: "azurerm_private_dns_private_zone_multiple",
```

Comment:
This is minor but shouldn't the folder nammed `azurerm_private_dns_zone_multiple` instead?

Reply:
yeah I don't think it could collide so I'll merge as is for now